### ISSUE TITLE
typo causing broken link redirection in spanish docs 

### ISSUE THREAD

**Issue by 1fbr** [expressjs/expressjs.com#1707]
### Description

On the **Spanish version** of the "Static Files" page ([https://expressjs.com/es/starter/static-files.html](https://expressjs.com/es/starter/static-files.html)), the "documentación en inglés" link points to an incorrect URL:

- **Incorrect URL**: [https://expressjs.com/en/starter/static-filen.html](https://expressjs.com/en/starter/static-filen.html)
- **Expected URL**: [https://expressjs.com/en/starter/static-files.html](https://expressjs.com/en/starter/static-files.html)

This typo ("**filen**" instead of "**files**") results in a 404 error.

---

### Supporting Evidence
1. Screenshot of the Spanish page:
   ![Spanish Page Issue](https://github.com/user-attachments/assets/d26127ee-0692-4e55-9117-e1a31f0d7c38)

2. Screenshot of the broken link:
   ![Broken Link](https://github.com/user-attachments/assets/d1a7fc1b-3013-46cb-af54-45273ea4d71f)

---

### Attempts to Debug
I tried searching for the typo `"filen"` in the repository but couldn't locate it. I guess the URL is being dynamically generated, but I have no clue on how to fix it.

---

### Expectations

Should redirect to this page: [https://expressjs.com/en/starter/static-files.html](https://expressjs.com/en/starter/static-files.html)

**Comment by bjohansebas** [comment]
Wow, that's very strange, that shouldn't happen

**Comment by 1fbr** [comment]
Wow, seeing your response, it gave me an idea, out of the blue, and I realised what's going on! It seems that when replacing the language code *es* with *en*, it’s matching the *es* in the filenames too.

To verify, I checked for similar cases, and the same issue occurs on pages like:

 https://expressjs.com/es/guide/behind-proxies.html
 https://expressjs.com/es/guide/using-template-engines.html

This pattern happens with any file whose name ends in *es*. So, yeah, it’s not a simple typo but a bug.

 I hope this helps @bjohansebas !
